core/stream/convert.go

```diff
@@ -399,9 +440,29 @@ func ftou(count int, dst, src buf) error {
 		}
 	case srcIsF64:
 		scale := float64(dstMask)
+		min, max := float64(math.MaxFloat64), float64(0)
+		for i := 0; i < count; i++ {
+			f := math.Float64frombits(ss2.Read(64))
+			if f < min {
+				min = f
+			}
+			if f > max {
+				max = f
+			}
+			ss2.ReadPos += src.stride - 64
+		}
 		for i := 0; i < count; i++ {
 			f := math.Float64frombits(ss.Read(64))
 			if norm {
+				if src.component.Channel == Channel_Depth {
+					f = (f - min) * (1 / (max - min))
+				} else if min < 0 {
+					if f <= 0 {
+						f = (f - min) * (0.5 / -min)
```

Comment:
possible division by zero

Reply:
Are you concerned about stability, because I already check if min is not zero two lines prior